index.html

```diff
@@ -2423,11 +2423,16 @@ <h3>Co-signed</h3>
               <td>Individual</td>
               <td>Development; open-source community efforts</td>
             </tr>
+		<tr>
+              <td><a href="https://github.com/fahadahammed">Fahad Ahammed</a></td>
+              <td>Individual</td>
+              <td>Testing;Documentation;open-source community efforts</td>
```

Comment:
Make sure to add spacing in the words here:
`<td>Testing; Documentation; open-source community efforts</td>`